### PR TITLE
Feature/deploy without eureka

### PR DESCRIPTION
**Pull request by ferrangrau**
@enriclluelles, @ismaelfernandezscmspain  I don't know if merge this or let in a branch. This PR adds the adds the feature of deploy without eureka.

### CODE REVIEW

**Comment by ferrangrau** [comment]
@ismaelfernandezscmspain @enriclluelles please! loock this PR I need add more changes...

File: deployer/deployer.py [hunk 1]
```diff
@@ -195,10 +201,209 @@ def deploy_version(self, version):
...
+
+        if good_deploy:
+            if remove_old:
+                for asg_name in self.get_asg_names_in_cluster():
+                    if asg_name != version:
+                        print("Deleting ASG {0}".format(asg_name))
+                        self.delete_asg(asg_name)
+        else:
+            self.disable_asg(asg_name=version)
+
```
Comment (ismaelfernandezscmspain) [inline deployer/deployer.py:278]: @ferrangrau Hi, I think that you can have other method, this check remove_old, the code more or less:

```
def check_remove_old_version(self, old_version):
       if old_version:
                 for asg_name in self.get_asg_names_in_cluster():
                     if asg_name != version:
                         print("Deleting ASG {0}".format(asg_name))
                         self.delete_asg(asg_name)
```

and in method "deploy_version_without_eureka":

```
     if good_deploy:
        check_remove_old_version(old_version)
     else:
         ...
```

**Comment by ismaelfernandezscmspain** [comment]
@ferrangrau One thing, the var `r` we have to change the name for request.

**Comment by ferrangrau** [comment]
@ismaelfernandezscmspain you are right, I do it. The r var is more to follow the convention in all the class but you are right again, the var name must be clear.

**Comment by ferrangrau** [comment]
@ismaelfernandezscmspain sorry, I forgot to push the change... can you review again?

**Comment by ismaelfernandezscmspain** [comment]
@ferrangrau Why you don't change the 'r' var in all code?
And... you don't forget to do test.

**Comment by ferrangrau** [comment]
@ismaelfernandezscmspain  The r I will change in another PR because I will change all the r not only for the methods of this PR, and you are right for the tests but I prefer standardize it in the stepper and then use the same way for all the modules.

**Comment by ismaelfernandezscmspain** [comment]
@ferrangrau I agree 
👍

**Pull request merged**